sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/nsexception/SentryUnhandledExceptions.kt

```diff
@@ -75,7 +75,7 @@ internal fun Throwable.asSentryEnvelope(): SentryEnvelope {
         hub.getClient()?.prepareEvent(event, hub.scope, alwaysAttachStacktrace = false, isCrashEvent = true)
     } ?: event
     val item = SentryEnvelopeItem(preparedEvent)
-    // TODO: pass traceState when enabling performance monitoring for KMP SDK
+    // TODOs: pass traceState when enabling performance monitoring for KMP SDK
```

Comment:
```suggestion
    // TODO: pass traceState when enabling performance monitoring for KMP SDK
```

Reply:
Detekt will flag TODO so one option would be to disable it, the other is to Suppress the line but that doesn't work since ktlint will format it in such way that the `@Suppress` will be in another line.

That aside though, I don't think we need this TODO, this specific envelope is only created during a crash and the purpose is only to send the crash event and won't be used for anything else.